Makefile

```diff
@@ -0,0 +1,39 @@
+# This rule runs the resource scaffolding script
+# $1 is the name of the resource to generate
+check_defined = \
+    $(strip $(foreach 1,$1, \
+        $(call __check_defined,$1,$(strip $(value 2)))))
+__check_defined = \
+    $(if $(value $1),, \
+      $(error Undefined $1$(if $2, ($2))))
+
+.PHONY: vet fmt lint test mocks envvars generate
+
+generate:
```

Comment:
When I run `make generate` it would be amazing if we could print a usage message instead of `Makefile:13: *** Undefined RESOURCE.  Stop.`